microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -737,7 +737,7 @@ Focus Mode is a new user interface for DevTools.  Focus Mode is designed to simp
 
 Focus Mode replaces the main row of tabs with an **Activity Bar**, which is a compact toolbar with distinctive icons.  The **Activity Bar** makes it possible to pin, rearrange, and open your favorite tools, for quick access.  The **Activity Bar** also provides access to user settings, help, and other features.
 
-Focus Mode also provides a **Quick View** list, to open a second tool alongside the tool that's already selected in the **Activity Bar**.
+Focus Mode also provides a **Quick View** panel at the bottom of DevTools, to open a second tool alongside the tool that's already selected in the **Activity Bar**.
 
 See [Reduce the complexity of DevTools with Focus Mode](focus-mode.md).
```

Comment:
I wonder why this isn't causing a build issue. I deleted this file from the big FM branch, since FM is no longer an experiment.
In fact, we should probably also remove this entire H2 section.

Reply:
@captainbrosset - IDK why this broken link isn't reported:
See [Reduce the complexity of DevTools with Focus Mode](focus-mode.md).
Anyway, I deleted that line/paragraph.  

In this PR, I'll delete the h2 section about Focus Mode checkbox.  
Monthly, I'll update the Experiments article per established process (except will be no longer tied to What's New PRs), and that will keep Experiments article's list of checkboxes up-to-date, in "main" branch.